{"type":"article_comment","id":94142,"parentType":"article","parentId":7296,"uid":17735,"contents":"好像是被 DDoS 了... Github 上刷 issue 的应该也是 Bot","date":"2019-10-30","agreeCount":2,"discussionCount":0}